presto-docs/src/main/sphinx/release/release-0.153.rst

```diff
@@ -5,6 +5,8 @@ Release 0.153
 General Changes
 ---------------
 
+* Fix bug in planner where incorrect results are generated when using grouping
```

Comment:
"Fix incorrect results for grouping sets when `task.concurrency` is greater than one."